### DISCUSSION TITLE
Two questions from a docker setup

### DISCUSSION THREAD

**Discussion by crawfordlong** [go-shiori/shiori#593] [Q&A] [answered]
Hi everyone,

I have a couple of questions I'm going to lump into one discussions post. If that's not ok, let me know, but my plan was to submit an issue if any or all of these aren't resolvable. 

I'm running shiori in a docker container on a server running ubuntu.

1. Migration from wallabag - I have downloaded the wallabag import script linked in the documentation, but it will not execute inside the container. The script is a bash script and the docker container seems (?) only to have sh available. 
2. Archiving - I have "create archive by default" **checked**, but when I add a link from the beta extension, it doesn't appear to be creating one. If I then click the archive button for the saved link, it pulls one mostly without an issue.

I'm brand new with shiori, so apologies if I've missed anything obvious.

**Comment by fmartingr** [comment, marked as answer]
Hey @crawfordlong, those two seem to be bugs. 

1) We updated the docker image a few months ago and `bash` is not included anymore. We could just include `bash` in the image for the scripting to work, which could be useful to run other scripts as well (since `sh` won't do, realistically).

2) This should not work like, what I think is happening is that the setting only works on the web server, rather than on the extension. On the other hand, there is an issue open on the web extension that claims that the extension always archives...? https://github.com/go-shiori/shiori-web-ext/issues/35

If you could open to issues about this thing I will try to fix the first one ASAP, the other may require more info and I'm not working much on the extension at the moment until I have the new API in place. Thanks for the reports, though!

**Comment by crawfordlong** [comment]
Fantastic, thank you.
